docs/content/python/getting-started/application/enforce.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add authorization enforcement
+description: |
+    Oso policies define rules for who can perform actions on resources
+    in your application. The Oso library queries the policy to obtain an
+    authorization decision — a yes or no answer. You enforce that
+    decision by rejecting or accepting requests that users make.
+weight: 2
+---
+
+# Add authorization enforcement
+
+## The allow rule
+
+At the end of [Model your authorization policy](model) we defined an `allow`
+rule. The `allow` rule is used for resource-level enforcement. It
+accepts three arguments: an `actor` (who is making the request), an
+`action` (what the actor wants to do), and a `resource` (the object that the
+actor wants to perform the action on).
+
+Here's the `allow` rule you wrote:
+
+```polar
+allow(actor, action, resource) if
+	has_permission(actor, action, resource);
+```
+
+This rule succeeds if the `actor` `has_permission` to perform `action` on
+`resource`. `actor`, `action`, and `resource` are all variables. They
+are provided when you query the policy.
+
+## Querying the policy
```

Comment:
I explained queries here because I feel it's necessary to get the connection between the policies and the app. I could just say the `authorize` method checks the allow rule to see if requests are authorized, but then in later guides when we fully explain querying and policy evaluation that could get confusing.